components/Buttons/src/MDCFloatingButton+Animation.m

```diff
@@ -89,110 +86,190 @@ + (float)fab_dragCoefficient {
 }
 #endif
 
++ (CFTimeInterval)fab_animationDurationFromWallTimeDuration:(CFTimeInterval)duration
+                                                   forLayer:(CALayer *)layer {
+  CFTimeInterval finalDuration = duration;
+
+#if TARGET_IPHONE_SIMULATOR
+  finalDuration *= [self fab_dragCoefficient];
+#endif
+
+  return [layer convertTime:finalDuration fromLayer:nil];
+}
+
++ (CFTimeInterval)fab_animationBeginTimeFromWallTimeOffset:(CFTimeInterval)offset
```

Comment:
A good follow-up change would be to change the "time offset" terminology here and throughout to the more standard "delay".

Reply:
sure